projects/js-packages/ai-client/src/chrome-ai/factory.ts

```diff
@@ -123,14 +124,39 @@ export default async function ChromeAIFactory( promptArg: PromptProp ) {
 		return chromeAI;
 	}
 
-	// TODO: consider also using ChromeAI for ai-assistant-summarize
 	if ( promptType.startsWith( 'ai-content-lens' ) ) {
+		if ( ! ( 'ai' in self ) || ! ( 'summarizer' in self.ai ) ) {
+			return false;
+		}
+
+		if ( context.language ) {
```

Comment:
This outer condition is not necessary, the inner condition is enough.

Reply:
Yep, this is the result of rewrite number 4 or 5. I'll change it.